internals/src/macros.rs

```diff
@@ -139,9 +139,9 @@ macro_rules! debug_from_display {
 /// Asserts a boolean expression at compile time.
 #[macro_export]
 macro_rules! const_assert {
-    ($x:expr) => {{
+    ($x:expr) => {
         const _: [(); 0 - !$x as usize] = [];
-    }};
+    };
```

Comment:
The reason is double parentheses made it possible to use it in non-item context (can't remember the proper name), so you've just disabled one usage in favor of the other. I'm not saying it's wrong, it's just different and if we don't use the other one then I don't mind but we might one day end up with two macros.